src/API.ts

```
@@ -49,7 +49,6 @@ class APIWrapper {
 				}))
 			}
 			catch (e) {
-				console.log(e)
```

Now that we know that you need to handle errors properly and that this interface will not be exposed to the application and will only be used for testing, we should consider throwing rather than returning empty arrays